fsharp-backend/src/LibExecution/RuntimeTypes.fs

```diff
@@ -845,10 +855,8 @@ type BuiltInFn =
     deprecated : Deprecation
     sqlSpec : SqlSpec
 
-    // TODO: reevaluate usefulness of below comment. at least rephrase.
-    // Functions can be run in JS if they have an implementation in this
-    // LibExecution. Functions whose implementation is in LibBackend can only be
-    // implemented on the server.
+    // Functions can be run in JS if they have an implementation in BackendOnlyStdlib.
```

Comment:
// Functions can be run in JS if they have an implementation in **LibExecution**.
    // Functions whose implementation is in **BackendOnlyStdLib** can only be implemented on the server.